lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+
+  def initialize
+    @meal = []
```

Comment:
Hmmm... not sure about `meal` as the name here. Could sound like an individual person's dinner or the food that everyone present will share.